### ISSUE TITLE
Add support for GCP e2-micro

### ISSUE THREAD

**Issue by tzach** [scylladb/scylla-machine-image#477]
GCP e2-micro will be used as non production, similar to EC2 t3-micro.
AFAIK, we use dev-mode for EC2 t3-micro, we can do the same here.

**Comment by mixellent** [comment]
Related to https://github.com/scylladb/scylla-machine-image/issues/238 as well

**Comment by benipeled** [comment]
As mentioned in #238 this type has no local SSD but if you are fine with this for "non-production" usage we can allow it and add a login-msg to make it cleat that it shouldn't be used (and supported) on production env.

@tzach @mixellent I'm curious, what's the planned usage for this instance on the cloud? testing env.?

**Comment by mixellent** [comment]
@benipeled yes it would be used as a sandbox only, similar to EC2 t3.micro. See what we would show in this case to the user:
![image](https://github.com/scylladb/scylla-machine-image/assets/969800/95143dd7-ed54-4260-94a5-0850cdef68ba)

**Comment by mykaul** [comment]
Can they even work? "e2-micro sustains 2 vCPUs, each for 12.5% of CPU time, totaling 25% CPU time and effectively consuming 0.25 cores." - with 2 cores mostly getting no CPU at all, how many cores will Scylla use (vs. the OS) ? Do we install (for example) the node_exporter there?

**Comment by tzach** [comment]
I successfully install and run ScyllaDB 5.2 (**in dev-mode**) in GCP e2-micro. All works as expected: scylla, nodetool, cqlsh.
